test-case/check-audio-equalizer.sh

```diff
@@ -18,13 +18,14 @@ set -e
 
 # source from the relative path of current folder
 my_dir=$(dirname "${BASH_SOURCE[0]}")
+# shellcheck source=case-lib/lib.sh
```

Comment:
This did get rid of a few shellcheck warnings (thx!) but fewer than I thought. Maybe this below would also help. Also off-topic sorry but also a tiny change/commit.

Warning: "latest" Ubuntu is not always the latest.

```diff
--- a/.github/workflows/pull_request.yml
+++ b/.github/workflows/pull_request.yml
@@ -19,7 +19,7 @@ on: [pull_request, workflow_dispatch]
 jobs:
 
   shellcheck:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@v4
         with:
@@ -32,7 +32,7 @@ jobs:
                text/x-shellscript shellcheck -x
 
   pylint:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     steps:
       # :-( https://github.community/t/support-for-yaml-anchors/16128
       - uses: actions/checkout@v4
```